app/src/main/java/com/sopt/now/compose/LoginActivity.kt

```diff
@@ -150,6 +160,32 @@ fun Login(signupId: String?, signupPw: String?, nickname: String?, mbti: String?
         )
         Button(
             onClick = {
+                val requestLoginDto = RequestLoginDto(id, pw)
+                authService.login(requestLoginDto).enqueue(object : Callback<ResponseSignupDto> {
```

Comment:
```suggestion
                authService.login(RequestLoginDto(id, pw)).enqueue(object : Callback<ResponseSignupDto> {
```

저는 변수 지정할 때 꼭 변수로 지정할 필요가 있을지 생각하고 선언하는 편입니닷